com.regnosys.rosetta.lib/src/main/java/model/annotations.rosetta

```diff
@@ -0,0 +1,9 @@
+namespace "com.rosetta.model"
+version "${project.version}"
+
+anno keyed:
```

Comment:
Can we change the keyword to `annotation`?

Reply:
Sure